internal/kv/constant.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2023 Linkall Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kv
+
+const (
+	keyCore     = "/vanus/core/"
+	keyResource = keyCore + "resource/"
+	keyExecutor = keyCore + "executors/"
+
+	// eventbus.
+	ResourceEventbus          = keyResource + "eventbus"
+	ResourceEventlog          = keyResource + "eventlog"
+	ResourceSegment           = keyResource + "segment"
+	ResourceSegmentOfEventlog = keyResource + "segment_eventlog"
+	ResourceVolume            = keyResource + "volume/"
```

Comment:
why have a suffix of "/"

Reply:
It isn't used in other packages, it needs to change to the inner variable and provide the function for other package use, I add issue #514 to resolve it